src/clr-core/styles/_fonts.scss

```diff
@@ -0,0 +1,29 @@
+// lite version of font should be used for core based applications
+// temporary paths until updated font for v3
+@font-face {
```

Comment:
Think we ought to be using our updated font, which I can make available.

Reply:
Updated to use the new package